.go-version

```diff
@@ -0,0 +1 @@
+1.14.12
```

Comment:
Nit: Should we go to 1.15.?

Reply:
This is matching beats. I think we should just map to beats, when beats moves we move.

Reply:
I think the nice part of being a separate binary is that we are not tied to what Beats has. I would rather tie it to the version the Elastic Agent uses if needed.